### DISCUSSION TITLE
Citra ?

### DISCUSSION THREAD

**Discussion by ShellInLove** [RedShyGuy/Vapecord-ACNL-Plugin#86] [Q&A]
Hello, is it possible to install your plugin on citra ? It's amazing on my 3ds but I cannot seem to find a way to install it on Citra.
Thank you in advance !

**Comment by RedShyGuy** [comment]
No the plugin is not working on Citra as it needs Luma to run

**Reply by ShellInLove** [reply]
Okay, thank you for answering!
